compiler/parser.nim

```diff
@@ -555,7 +556,7 @@ proc parsePar(p: var TParser): PNode =
   optInd(p, result)
   flexComment(p, result)
   if p.tok.tokType in {tkDiscard, tkInclude, tkIf, tkWhile, tkCase,
-                       tkTry, tkDefer, tkFinally, tkExcept, tkFor, tkBlock,
+                       tkTry, tkDefer, tkFinally, tkExcept, tkBlock,
```

Comment:
I'm confused, but why is `tkFor` removed here?

Reply:
Occurs 2 times in this set, simple refactoring

Reply:
Oh, I'm blind, sorry.